package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@txo/functional",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
it should be 1.3.0 you are adding new functionality.

Reply:
updated